app/views/configure/archiving.phtml

```diff
@@ -47,40 +30,91 @@
+					<?php echo _t('conf.archiving.retention_period'); ?>
+					<input type="number" id="retention_period_count" name="retention_period_count" min="0" value="<?php echo FreshRSS_Context::$user_conf->volatile['retention_period_count']; ?>" data-leave-validation="<?php echo FreshRSS_Context::$user_conf->volatile['retention_period_count']; ?>"/>
+					<select class="number" name="retention_period_unit" id="retention_period_unit" data-leave-validation="<?php echo FreshRSS_Context::$user_conf->volatile['retention_period_unit']; ?>">
+						<option></option>
+						<option value="P1Y" <?php echo 'P1Y' === FreshRSS_Context::$user_conf->volatile['retention_period_unit'] ? 'selected="selected"' : '' ?>><?php echo _t('gen.period.years'); ?></option>
+						<option value="P1M" <?php echo 'P1M' === FreshRSS_Context::$user_conf->volatile['retention_period_unit'] ? 'selected="selected"' : '' ?>><?php echo _t('gen.period.months'); ?></option>
+						<option value="P1W" <?php echo 'P1W' === FreshRSS_Context::$user_conf->volatile['retention_period_unit'] ? 'selected="selected"' : '' ?>><?php echo _t('gen.period.weeks'); ?></option>
+						<option value="P1D" <?php echo 'P1D' === FreshRSS_Context::$user_conf->volatile['retention_period_unit'] ? 'selected="selected"' : '' ?>><?php echo _t('gen.period.days'); ?></option>
+						<option value="PT1H" <?php echo 'PT1H' === FreshRSS_Context::$user_conf->volatile['retention_period_unit'] ? 'selected="selected"' : '' ?>><?php echo _t('gen.period.hours'); ?></option>
+					</select>
+				</label>
+			</div>
+		</div>
+
+		<div class="form-group">
+			<div class="group-controls">
+				<label class="checkbox" for="keep_favourites">
+					<input type="checkbox" name="keep_favourites" id="keep_favourites" value="1"<?php echo FreshRSS_Context::$user_conf->archiving['keep_favourites'] ? ' checked="checked"' : ''; ?>  data-leave-validation="<?php echo FreshRSS_Context::$user_conf->archiving['keep_favourites']; ?>"/>
+					<?php echo _t('conf.archiving.keep_favourites'); ?>
+				</label>
+			</div>
+		</div>
 
 		<div class="form-group">
-			<label class="group-name"><?php echo _t('conf.user.current'); ?></label>
 			<div class="group-controls">
-				<?php echo _t('conf.user.articles_and_size', format_number($this->nb_total), format_bytes($this->size_user)); ?>
+				<label class="checkbox" for="keep_labels">
+					<input type="checkbox" name="keep_labels" id="keep_labels" value="1"<?php echo FreshRSS_Context::$user_conf->archiving['keep_labels'] ? ' checked="checked"' : ''; ?>  data-leave-validation="<?php echo FreshRSS_Context::$user_conf->archiving['keep_labels']; ?>"/>
+					<?php echo _t('conf.archiving.keep_labels'); ?>
+				</label>
 			</div>
 		</div>
 
-		<?php if (FreshRSS_Auth::hasAccess('admin')) { ?>
 		<div class="form-group">
-			<label class="group-name"><?php echo _t('conf.user.users'); ?></label>
 			<div class="group-controls">
-				<?php echo format_bytes($this->size_total); ?>
```

Comment:
Do we see this information elsewhere now?

Reply:
In the user page when you select a user in the delete drop-down. I want to add that as well in #2417. 
I cannot think of any good reason to have this information here.

Reply:
It's back on the menu :)